### DISCUSSION TITLE
How to enable vPMU.

### DISCUSSION THREAD

**Discussion by zephr4** [opcm/pcm#437] [Q&A]
Hi,

I'm trying to run pcm via the docker image, found on hub.docker.com, on a Lenovo ThinkStation P350 running Windows 10 Pro. When I run the image, I come across an error stating that vPMU needs to be enabled on the hypervisor. How can I enable this setting to move forward?

The following is the error:
```
Detected a hypervisor/virtualization technology. Some meterics might not be available due to configurations or availability of virtual hardware features.

=====  Processor information  =====
Linux arch_perfmon flag  : no
ERROR: vPMU is not enabled in the hypervisor. Please see details in https://software.intel.com/content/www/us/en/develop/documentation/vtune-help/top/set-up-analysis-target/on-virtual-machine.html
       you can force-continue by setting PCM_IGNORE_ARCH_PERFMON=1 environment variable.
ERROR: Can not open /proc/sys/kernel/nmi_watchdog file.
```

**Comment by opcm** [comment]
the container runs inside WSL. WSL on Windows 10 does not expose vPMU. If I correctly remember recent versions of WSL on Windows 11 expose vPMU but this has not been tested with PCM
